components/_dropdowns.scss

```diff
@@ -25,7 +29,9 @@ $dropdown-content-z-index: 1;
 	background-color: $color-white;
 	box-shadow: 0 0 10px 0 rgba($color-black, .2);
 	display: none;
+	left: 0;
 	position: absolute;
+	top: calc(100% + 1px);
```

Comment:
isn't nav specific style?
I'm not sure if it should be part of dropdown component.